scripts/install_selinux_venv.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+VENV_LIB_PATH=lib/python2.7/site-packages
+LIBSELINUX_PATH=/usr/lib64/python2.7/site-packages
+
+if [ -n "${VIRTUAL_ENV}" ]; then
```

Comment:
The dnf command here assumes root user. Why not ask if the user is root, and if not, prepend sudo onto the command? I explained why this would be useful above?